src/yaml/updateLayerMetadata.yaml

```diff
@@ -93,28 +93,28 @@ components:
           maximum: 8000
           format: double
           description: max (rounded) resolution of layer in meters/pixel
-        accuracyCE90:
+        minHorizontalAccuracyCE90:
           type: number
-          description: Horizontal Accuracy
-        sensorType:
+          minimum: 0.01
+          maximum: 4000
+          format: double
+          description: minimum Horizontal Accuracy
+        sensors:
           type: array
           items:
             type: string
-            enum:
-              - VIS
-              - RGB
-              - Pan_Sharpen
-              - OTHER
-              - UNDEFINED
-          description: The sensor type
+          description: Layer sensors list 
         region:
-          type: string
-          description: Resolution of tiff files
+          type: array
+          items:
+            type: string
+          description: regions included in the layer
         rms:
           type: number
         scale:
-          type: string
-          pattern: "^(0|[1-9]\\d{0,8})$"
+          type: integer
+          minimum: 0
+          maximum: 100000000
```

Comment:
same validation is missing on field decoration level ( see an example in resolution decoration)